src/main/java/io/gravitee/repository/mongodb/utils/FieldUtils.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.repository.mongodb.utils;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * @author Yann TAVERNIER (yann.tavernier at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public class FieldUtils {
+
+    public static String toCamelCase(String field) {
+
+        if (StringUtils.isEmpty(field)) {
+            return field;
+        }
+
+        String result = field;
+        while(result.contains("_")) {
```

Comment:
Not optimized for string with more than 2 '_'. You should add a warning in comment about this.
We could use regex in these cases.